tests/subsys/sensing/boards/native_sim.overlay

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2024 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/sensing/sensing_sensor_types.h>
+
+&i2c0 {
+	bmi160_i2c: bmi@68 {
+		compatible = "bosch,bmi160";
+		reg = <0x68>;
+	};
+};
+
+&spi0 {
+	bmi160_spi: bmi@3 {
+		compatible = "bosch,bmi160";
+		spi-max-frequency = <50000000>;
+		reg = <0x3>;
+	};
+};
+
+/ {
+	sensing: sensing-node {
+		compatible = "zephyr,sensing";
+		status = "okay";
+
+		base_accel_gyro: base-accel-gyro {
+			compatible = "zephyr,sensing-phy-3d-sensor";
+			status = "okay";
+			sensor-types = <SENSING_SENSOR_TYPE_MOTION_ACCELEROMETER_3D SENSING_SENSOR_TYPE_MOTION_GYROMETER_3D>;
```

Comment:
@teburd this seems in conflict with the sensor metadata yaml that we discussed in the sensor working group. I should have all that code ready next week I think.

Reply:
This isn't adding any additional bindings though, the bindings already exist? I don't think blocking this PR over future work makes sense, we cannot constantly wait on what will be to do what needs to be done now. Tests are needed for this subsys and this is working towards that.